msfdb

```diff
@@ -12,6 +12,10 @@ require 'rex/text'
 require 'securerandom'
 require 'uri'
 require 'yaml'
+require 'pg'
+require_relative 'msfdb_helpers/pg_ctlcluster'
```

Comment:
I wonder if these should live somewhere in `lib`? Adding a custom top level folder stuck out to me as being a smell 🤔

Reply:
I'm not against putting it in `lib` can do that for sure
is there something _wrong_ with putting things in their own top level folder though? just for future reference